### ISSUE TITLE
Pin deployments to release tags

### ISSUE THREAD

**Issue by shenanigansd** [vipyrsec/infrastructure#34]
Now that we're starting to get manageable releases out for our apps, we should start pinning the deployments to the releases to avoid surprises.

**Comment by Robin5605** [comment]
Good idea. However the recommended way is to generally pin to commit hash (see [Security Hardening for GitHub Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)) Though this is mostly for third party actions, it wouldn't hurt to be consistent.

I don't think pinning to commit hashes for our own releases is too big of a deal but perhaps something worth discussing.

**Comment by shenanigansd** [comment]
> Good idea. However the recommended way is to generally pin to commit hash (see [Security Hardening for GitHub Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions))

https://github.com/vipyrsec/infrastructure/pull/35/commits/958484a53746a801e2c9aad99a2c86e5874abd6b
